pyqt_sql_demo/connection/model.py

```diff
@@ -2,17 +2,22 @@
 from PyQt5.QtCore import pyqtSignal
 
 import pyqt_sql_demo.connection.exceptions as exceptions
+import pyqt_sql_demo.widgets.text as UI
+
+from PyQt5.QtWidgets import QApplication, QInputDialog, QLineEdit
```

Comment:
Ideally, none of the UI libraries should be here. This module is only responsible for the interaction with the database and any changes to the user interface should be reflected in the different module.

Reply:
See below about requesting password